template-parts/global-templates/hero-pagetitle.php

```diff
@@ -50,16 +77,16 @@
 			$thumbnail_post_id = get_option( 'page_for_posts' );
 		}
 		$thumbnail_size = 'hero';
-		if ( $hero_style == 'img-under-title' && ! is_customize_preview() ) {
+		if ( $hero_style == 'img-under-title' || 'img-above-title' && ! is_customize_preview() ) {
 			$thumbnail_size = 'large';
 		}
 		if ( is_singular() && has_post_thumbnail( $thumbnail_post_id ) ) {
 			?>
-			<div class="hero-image-holder hero-featured-image-holder" data-aos="fade" data-aos-duration="1000">
+			<div class="hero-image-holder hero-featured-image-holder hero-image-bottom" data-aos="fade" data-aos-duration="1000">
 				<?php echo get_the_post_thumbnail( $thumbnail_post_id, $thumbnail_size ); ?>
 			</div>
 			<?php
-		} elseif ( ( $hero_style != 'img-under-title' || is_customize_preview() ) && $hero_default_images = get_theme_mod( 'hero_main_default_images' ) ) {
+		} elseif ( ( $hero_style != 'img-under-title' || 'img-above-title' || is_customize_preview() ) && $hero_default_images = get_theme_mod( 'hero_main_default_images' ) ) {
```

Comment:
This will always return true. I guess `$hero_style !=` is missing before `'img-above-title' ||`